webapp/src/components/Analysis/PreviewCard.tsx

```diff
@@ -6,10 +6,17 @@ import { Link } from "react-router-dom";
 type Props = {
   title: string;
   to?: string;
+  linkButton?: string;
   description: React.ReactElement<typeof Description>;
 };
 
-const PreviewCard: React.FC<Props> = ({ title, to, children, description }) => (
+const PreviewCard: React.FC<Props> = ({
+  title,
+  to,
+  linkButton,
```

Comment:
Since `linkButton` is optional, it should default to something. An empty button looks very weird.
![image](https://user-images.githubusercontent.com/8386369/187339562-30fba478-5b2d-4d31-bbca-6732b40ec763.png)
I suggest leaving `"View details"` as the default and removing it where it is specified in `Dashboard`. Also, can we come up with a better name? Maybe `linkButtonText` or simply `buttonText`? What do you think?
```suggestion
  buttonText = "View details",
```

Reply:
I agree with the appropriate name change. But I am curious to know where is the empty button rendered.

Reply:
Nowhere. I tested it locally.

Reply:
You are right, the empty link button just flashes in a second before text renders. So, it would be right to set the default value and we don't have to pass every time "View details" text.